src/main/java/objects/Player.java

```diff
@@ -1,25 +1,61 @@
 package objects;
 
-import com.sun.org.apache.xpath.internal.operations.Bool;
 import constants.Constant;
```

Comment:
Lets shorten these imports.

```suggestion
import constants.*;
```